docs/README.md

```diff
@@ -8,3 +8,39 @@ jobs that the bootstrapper does:
 - Run the kubelet as a windows service
 
 This will be remotely invoked from a Ansible script or can be run locally
+
+## Requirements
+
+- Must be run on Windows server 2019
+- Must be run as administrator
+- A worker ignition file generated by the cluster must be on disk
+- The kubelet you wish to use must be on disk
+
+##Usage
```

Comment:
The missing space after `##` is causing this not to be rendered correctly. Please fix.